### DISCUSSION TITLE
[BUG] WebUI doesnt show proper time

### DISCUSSION THREAD

**Discussion by UltraHKR** [AnalogJ/scrutiny#883] [Issue Triage]
**Describe the bug**
The temp graph doesnt show the proper (adjusted to timezone) time.
I use "America/Guayaquil", my local timezone, but temp graph keeps showing GMT time.

**Expected behavior**
HDD last update time should match the temp graph loosely........

**Screenshots**
![image](https://user-images.githubusercontent.com/49404533/182401177-6763518e-12b6-4ad5-b9a5-5b0fd8c89176.png)

[docker compose..yml]
---
version: "3.3"
services:
  scrutiny:
    image:   ghcr.io/analogj/scrutiny:master-omnibus
    container_name: scrutiny
    environment:
      - TZ=America/Guayaquil
      - COLLECTOR_CRON_SCHEDULE="*/15 * * * *"
    cap_add:
      - SYS_RAWIO
      - SYS_ADMIN
    volumes:
      - /run/udev:/run/udev:ro
      - /mnt/V3700-JBOD/Docker_Data/SMART.Scrutiny:/scrutiny/config
      - /mnt/V3700-JBOD/Docker_Data/SMART.Scrutiny/influxdb:/opt/scrutiny/influxdb
    ports:
      - 8079:8080
      - 8078:8086
    restart: unless-stopped
    devices:
      - "/dev/sdb"
...
      - "/dev/sdy"

The log files will be available on your host in the `config` directory. Please attach them to this issue. 
(There are NO log files there....)
docker logs scutiny:
[smartscrutiny.log.txt](https://github.com/AnalogJ/scrutiny/files/9243130/smartscrutiny.log.txt)

Please also provide the output of `docker info`

Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Docker Buildx (Docker Inc., v0.7.1-docker)
  scan: Docker Scan (Docker Inc., v0.12.0)

Server:
 Containers: 26
  Running: 21
  Paused: 0
  Stopped: 5
 Images: 22
 Server Version: dev
 Storage Driver: zfs
  Zpool: V3700-JBOD
  Zpool Health: ONLINE
  Parent Dataset: V3700-JBOD/Docker_Data
  Space Used By Parent: 29105191440
  Space Available: 4520819491888
  Parent Quota: no
  Compression: lz4
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7b11cfaabd73bb80907dd23182b9347b4245eb5d
 runc version: v1.0.2-0-g52b36a2
 init version: de40ad0
 Security Options:
  apparmor
 Kernel Version: 5.10.120+truenas
 Operating System: Debian GNU/Linux 11 (bullseye)
 OSType: linux
 Architecture: x86_64
 CPUs: 6
 Total Memory: 47.06GiB
 Name: truenas
 ID: AZO6:CTLV:DKMN:7HZV:TIEH:AR2D:JKSF:BACB:SEOD:55D2:KXLE:SHJT
 Docker Root Dir: /mnt/V3700-JBOD/Docker_Data
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

**Comment by AnalogJ** [comment]
Good catch, I'll take a look at this. Local vs Remote timezone issues are usually kind of annoying to debug, so I can't guarantee that this will be fixed quickly, but it's definitely on the list now. 

Thanks!

**Comment by UltraHKR** [comment]
maybe a temporal compromise could be treating user defined timezone as "local time"?

**Comment by UltraHKR** [comment]
I'm sorry why is this being closed?

**Reply by kaysond** [reply]
Can you please try v0.8.2?

**Reply by Pheoxy** [reply]
Bumped image on my NixOS server to v0.8.2 and it still has UTC as time.
Although the Drive cards seem to show time properly?

<img width="1406" height="836" alt="image" src="https://github.com/user-attachments/assets/974440d0-2547-4c07-8c2a-56e7d2ffc58a" />

```
{ ... }:
{
  virtualisation.oci-containers.containers.scrutiny = {
    autoStart = true;
    hostname = "scrutiny";
    networks = [ "scrutiny" ];
    image = "ghcr.io/analogj/scrutiny:v0.8.2-omnibus";
    environment = {
      TZ = "Australia/Perth";
    };
    # Keep web UI closed by default; uncomment when interactive UI access is needed.
    # ports = [
    #   "18080:8080/tcp"
    # ];
    volumes = [
      "/srv/services/nimue/scrutiny/config:/opt/scrutiny/config"
      "/srv/services/nimue/scrutiny/influxdb:/opt/scrutiny/influxdb"
      "/run/udev:/run/udev:ro"
    ];
    devices = [
      "/dev/disk/by-id/ata-WDC_WD101EFBX-68B0AN0_VCHY9XXP"
      "/dev/disk/by-id/ata-WDC_WD101EFBX-68B0AN0_VCHZNRMP"
      "/dev/disk/by-id/ata-WDC_WD101EFBX-68B0AN0_VCHZW0DP"
      "/dev/disk/by-id/ata-WDC_WD101EFBX-68B0AN0_VCHZX30P"
      "/dev/disk/by-id/ata-WDC_WD101EFBX-68B0AN0_VCHZX66P"
      "/dev/disk/by-id/ata-WDC_WD101EFBX-68B0AN0_VCHZYWDP"
      "/dev/disk/by-id/ata-Samsung_SSD_860_EVO_1TB_S3YBNB0JC05733J"
      "/dev/disk/by-id/ata-Samsung_SSD_850_EVO_500GB_S2R9NB0HA08935K"
      "/dev/nvme0:/dev/nvme0"
      "/dev/nvme0n1:/dev/nvme0n1"
    ];
    extraOptions = [
      "--cap-add=SYS_RAWIO"
      "--cap-add=SYS_ADMIN"
    ];
  };
}
```

**Comment by kaysond** [comment]
Dupe of #889
